docs/strategy-customization.md

```diff
@@ -249,6 +249,22 @@ minimal_roi = {
 
 While technically not completely disabled, this would sell once the trade reaches 10000% Profit.
 
+To use times based on candle duration (ticker_interval or timeframe), the following snippet can be handy.
+This will allow you to change the ticket_interval for the strategy, and ROI times will still be set as candles (e.g. after 3 candles ...)
+
+``` python
+from freqtrade.exchange import timeframe_to_minutes
+
+class AwesomeStrategy(IStrategy):
+
+    ticker_interval = '1d'
+    ticker_interval_mins = timeframe_to_minutes(ticker_interval)
+    minimal_roi = {
+        str(ticker_interval_mins * 3)): 0.02,  # After 3 candles
+        str(ticker_interval_mins * 6)): 0.01,  # After 6 candles
+    }
```

Comment:
My suggestion (can support @yazeed here, it's not needed, but better be added to avoid questions):

```
    minimal_roi = {
        '0' : 0.03,                            # 3% till 3 candles
        str(ticker_interval_mins * 3)): 0.02,  # 2% after 3 candles
        str(ticker_interval_mins * 6)): 0.01,  # 1% after 6 candles
    }
```